<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Amigos</class>
 <widget class="QDialog" name="Amigos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>1158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="logo.qrc">:/imagenes/music.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>841</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>7</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font: Consolas; font-size:25pt; color:#ff4f52;&quot;&gt;Amigos con mismos gustos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="Framebusqueda">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>811</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#091a31;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>721</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffffff;</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="Buttonbox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>167</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffffff;
color:#000000;</string>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>470</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="logo.qrc">:/imagenes/music.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_seguidores">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>580</y>
     <width>291</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_seguidos">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>590</y>
     <width>321</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_seguir">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>550</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>550</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ingresar id usuario</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seguir">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>550</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>seguir</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
